src/pip/_internal/wheel.py

```diff
@@ -1091,26 +1094,22 @@ def build(
             'Building wheels for collected packages: %s',
             ', '.join([req.name for (req, _) in buildset]),
         )
-        _cache = self.wheel_cache  # shorter name
+
+        python_tag = None
+        if should_unpack:
+            python_tag = pep425tags.implementation_tag
+
         with indent_log():
             build_success, build_failure = [], []
-            for req, ephem in buildset:
-                python_tag = None
-                if should_unpack:
-                    python_tag = pep425tags.implementation_tag
-                    if ephem:
-                        output_dir = _cache.get_ephem_path_for_link(req.link)
-                    else:
-                        output_dir = _cache.get_path_for_link(req.link)
-                    try:
-                        ensure_dir(output_dir)
-                    except OSError as e:
-                        logger.warning("Building wheel for %s failed: %s",
-                                       req.name, e)
-                        build_failure.append(req)
-                        continue
-                else:
-                    output_dir = self._wheel_dir
+            for req, output_dir in buildset:
+                try:
+                    ensure_dir(output_dir)
+                except OSError as e:
+                    logger.warning("Building wheel for %s failed: %s",
```

Comment:
nit: since we're moving this bit of code, it'd be nice to make the indent/parenthesis style consistent with the surrounding code:

```py
logger.warning(
	"...",
	..., ...
)
```

Reply:
Done [here](https://github.com/pypa/pip/pull/6978/files#diff-79304b3f74b6bf37043da4d96ccd3755R1109-R1112).